external/js/cothority/README.md

```diff
@@ -49,6 +49,37 @@ You must use the given script instead of `npm publish` because we need to publis
 the _dist_ folder instead. If you try to use the official command, you will get
 an error on purpose.
 
+### Message classes
+
+You can write a class that will be used when decoding protobuf messages by using
+this template:
+```javascript
+class MyMessage extends Message<MyMessage> {
+    /**
+     * @see README#Message classes
```

Comment:
I don't know if you expected this to work as a link, but it doesn't.

Reply:
No I don't think there is an actual way to do that. I'm open to suggestions.